### ISSUE TITLE
bug

### ISSUE THREAD

**Issue by luisfranjoso** [dionpirotta/Custom-CSGO-HUD#2]
First of all, excelent hud, congrats!

it has a little bug with bomb and defuse timer.
https://clips.twitch.tv/KawaiiFitWaspMoreCowbell

**Comment by dionpirotta** [comment]
What is the bug specifically? I dont see what you are referring to?

**Comment by forztk** [comment]
The timer from defuse and bomb gives the feeling that it can be defused in time. Because the defuse bar(blue) looks smaller than the bomb bar (red).

**Comment by zahti** [comment]
I agree that the defuse vs bomb explosion timers should probably be updated / modified. @dionpirotta what files would need to be modified to make these changes? I could play around with it today!
